lib/validation.js

```diff
@@ -18,10 +18,12 @@
  *
  * @module numberValidation
  */
+const isDecimal = require('validator/lib/isDecimal');
+const isInt = require('validator/lib/isInt');
 
-module.exports = function numberValidation(input, settings) {
-  if (input.target.value === '' && !settings.target.empty) {
-    return `${input.target.name} cannot be left blank!`;
+module.exports = function numberValidation(input) {
+  if (input.target.value && !isInt(input.target.value.toString()) && !isDecimal(input.target.value.toString())) {
```

Comment:
It needs to check to see that there _is_ a value and that the value _is either_ an integer _or_ a decimal